### DISCUSSION TITLE
Un logo ?

### DISCUSSION THREAD

**Discussion by uurshin** [fibreville/minotaure_php#83] [Boite à idées]
Happy1000 et Seikame, il me semble que vous êtes dans des métiers qui touchent au dessin. Est-ce que ça vous tenterait de contribuer au projet en faisant un logo / un favicon ? Moi j'ai quelques connaissances mais rien de bien poussé, du coup si on a un pro là dessus ce serait génial.

**Comment by Seikame** [comment]
Let's go ! Déjà est-ce qu'on est sûrs de rester sur le nom ATRPG ? Moi il me va mais y'avait un doute à un moment de ta part si je ne m'abuse. Pas forcément besoin que le nom apparaisse dans le coeur du logo de toute façon.

J'initie la tempête de cerveau, mais je rappelle que les logos c'pas ma spécialité, vous moquez pas. x)

En axes qui pourraient être évoqués on a quoi, moi je vois surtout : 
- le côté chat de discussion
- le format streaming/diffusion en direct, sans forcément mettre l'accent sur twitch j'imagine
- le côté aléatoire des épreuves et des tirages au sort

Ptet une silhouette de streamer/streameuse avec un ou deux éléments identifiables (bras-micro, casque audio...) qui plane au dessus de petits avatars assez peu caractérisés (qui font éventuellement n'importe quoi) placés sur un socle en relief qui rappellerait un camembert de sondage ?

Pour l'instant c'est ça ou je re-propose des trucs bizarres entre le dragon d'Opale et l'avatar de Fibre. :D

**Reply by ElethornGM** [reply]
le dragon d'opale qui tient l'avatar de fibre dans sa main ? Et dans chaque oeil de l'avatar de fibre, un dragon d'opale roliste ! Oh mon dieu !
Moi j'y connais pas grand chose en logos, donc je vous laisse là dessus, mais je suivrais la conversation pour si jamais :)

**Reply by uurshin** [reply]
Je préfère séparer Opale du projet ATRPG car :
- ma contribution se fait à titre personnel, pas au titre de l'asso
- le code de base n'a pas de rapport avec Opale, et même si je suis le contributeur principal des nouvelles version ce n'est pas mon projet perso
- le seul lien que j'aimerais garder avec Opale c'est la possibilité de déclarer des parties là bas (et c'est déjà fait)

Donc si on peut réfléchir à un logo totalement détaché d'Opale Rôliste et même de FibreTigre c'est le mieux. Après tout un logo doit surtout faire penser aux possibilités qu'apporte le logiciel. J'aime bien les pistes que tu as donné Seikame, je verrai si je peux réfléchir à un mockup de mon côté à partir de ça, j'aurai ptet des idées complémentaires.

**Reply by HappyFR** [reply]
Je me posais la même question concernant le nom ATRPG. Après peut-être que Fibre a son mot a dire là dessus vu que c'est son système?  Mais je pense que c'est clair pour personne atrpg en tout cas XD.

Après pour le style  du logo en lui même, je pense qu'il faudrait quelques chose d'assez simple pour être dans l'ère du temp. Un peu comme le logo de Gartic Phone. Je suis pas un spécialiste des logos mais avant de tester des choses il faudrait être caler sur le nom et quels visuels on pourrait tester. 
Trouver une idée qui évoque que c'est en ligne + que c'est du jdr + que c'est multi ça va être un gros défi je pense. Si on arrive a trouver une idée simple ce serait cool, mais j'ai pas de proposition là de suite.

**Reply by vouiv-re** [reply]
J’aimerais apporter ma contribution ^^
Je ne suis pas un expert en logo, mais je fait un peu de vectoriel, je me dit que ça pourrait aider.

* Je trouve qu'il n'y a rien de mieux que le dé à 20 faces pour symboliser le JDR, ou à défaut n'importe quel dé
* Il faut symboliser le côté multijoueur, parce que c'est qui rend l'expérience intéressante (Je pense à la fameuse icône [users](https://fontawesome.com/icons/users?style=solid))
* Et enfin le micro de table genre [microphone](https://fontawesome.com/icons/microphone?style=solid) pour représenter l’expérience de la narration

Loin de moi l'envie de faire de la pub pour font-awesome, les icônes sont juste la à titre d'illustration. Le logo c'est un l'expression visuelle d'un concept et je pense qu'il faut savoir utiliser des poncifs du genre pour exprimer un concept clair : 

* Un JDR massivement multijoueur

**Comment by twaiiiin** [comment]
Bonjour,

Je profite de ce fil de discussion pour parler du nom ATRPG. Selon moi, "Asynchronous Tactical Role-Playing Game" ne fait pas sens par rapport au système de jeu.

Asynchronous / Asynchrone : Ce système de jeu n'est pas asynchrone puisque le MJ et les joueurs doivent être tous connectés et jouer en même temps lors d'une diffusion live en stream. Par contre, il est clairement asymétrique, puisque le MJ et les joueurs ne jouent pas de la même manière. Et certaines joueurs ont des pouvoirs spécifiques (leader & traitre).

Tactical Role-Playing Game / Jeu de rôle tactique : Le RPG tactique est un sous-genre spécifique de jeu stratégique au tour par tour où les joueurs planifient leurs déplacements et leurs attaques sur une grille (souvent en 3D isométrique). Ici, on n'a ni déplacement, ni attaque, ni "tour" à proprement parlé, ni représentation graphique des joueurs. Cela ne fait pas vraiment sens.

Pour moi, ce moteur de jeu est plus proche de la mouvance récente du crowd play / crowd choice que du jeu de rôle tactique et asynchrone. Qu'est-ce que vous en pensez ?

**Reply by vouiv-re** [reply]
Bonjour !

Je trouve ton sujet très intéressant, je m'étais fait la même réflexion, bien qu'avec moins  de finesse^^ Il me semble qu'un fil dédié serait peut-être plus adapté pour ce sujet, qu'en pense @uurshin ?

Il ne faut pas oublier non plus qu'un nom de domaine à été pris. Alors bien sur il n'y a pas forcément de corrélation entre ce repo et un éventuel hébergement, mais bon ça pourrait être une contrainte.

Après s'il faut tripatouiller un nom pour retomber sur les des lettres précises, je serais ravi de contribuer.

**Reply by uurshin** [reply]
@vouiv-re hello ! Je pense qu'on peut continuer à en discuter ici car si le nom change, le logo va surement se retrouver influencé (et ça va libérer des pistes de réflexions).
Pour les réflexions de @twaiiiin sur le nommage, déjà bienvenue et merci pour ton intervention ! J'en avais parlé un peu avec les autres dev et streamers mais je n'ai pas poussé plus loin pour ne pas donner l'impression de vouloir faire table rase de tout, vu que j'avais déjà bien chamboulé le code.

Un nom c'est effectivement très important, et je serai tout à fait pour en trouver un autre. On a néanmoins quelques contraintes, comme l'évoque Vouivre :
- le nom de ce repo (facile à changer @cell ?)
- la communication déjà faites autour (mais bon on débute donc c'est pas forcément grave)
- le nom de domaine qu'on a pris avec un petit groupe de streamers (Happy1000, Seikame, Elethorn, moi et d'autres à venir). Comme le dit bien Vouivre, ce groupe n'est pas représentatif de ATRPG dans son ensemble, nous ne sommes que des utilisateurs parmi d'autres, mais mine de rien toutes les personnes qui le diffusent actuellement sont dessus, donc c'est un peu dommage. Ce n'est pas gravissime non plus !
- l'accord de FibreTigre pour changer le nom, par principe

L'idée de trouver un autres sens aux lettres de ATRPG est intéressant mais je ne sais pas si on va s'en sortir sans virer au délire 😄 .

**Reply by twaiiiin** [reply]
> * le nom de ce repo (facile à changer @cell ?)

Un nom de repo Github, ça se change facilement.

> * l'accord de FibreTigre pour changer le nom, par principe

Ce point me semble essentiel. Mon message précédent est le résultat de beaucoup d'interrogations que je me pose depuis que FibreTigre a diffusé la première version de son application. Pour aller plus loin, je serais même curieux et intéressé de connaitre son avis et son point de vue de Game Designer sur le nom ATRPG. Qu'est-ce qui l'a motivé à appeler l'application comme cela ? Je ne suis pas expert dans ces sujets et il y a peut-être une explication que je n'ai pas envisagé.

> * la communication déjà faites autour (mais bon on débute donc c'est pas forcément grave)
> * le nom de domaine qu'on a pris avec un petit groupe de streamers (Happy1000, Seikame, Elethorn, moi et d'autres à venir). Comme le dit bien Vouivre, ce groupe n'est pas représentatif de ATRPG dans son ensemble, nous ne sommes que des utilisateurs parmi d'autres, mais mine de rien toutes les personnes qui le diffusent actuellement sont dessus, donc c'est un peu dommage. Ce n'est pas gravissime non plus !

C'est deux points sont davantage problématiques, en effet.

**Reply by cell** [reply]
:+1: pour la possibilité de renommer un répo (il y a même une redirection faite par github pendant un mois).

**Reply by cell** [reply]
Par contre, ca reste un hors sujet le changement de nom :wink:

**Reply by uurshin** [reply]
Pas vraiment hors sujet, les deux sont très liés. Si on envisage de changer de nom ça impacte le logo.

**Comment by vouiv-re** [comment]
Hello !
J'ai une proposition pour le nom (et accessoirement pour le logo).

Je me suis donné 4 contraintes :

* Faire un [acronyme récursif](https://fr.wikipedia.org/wiki/Sigle_r%C3%A9cursif) : Parce que je suis un peu malade et que j'aime ça -_-'
* Utiliser un nom commun : Parce que ça se retiens et ça s'écrit facilement
* Trouver un nom de domaine libre qui lui correspond
* Le terminer par "RE" parce que le ".re" est utilisable en Europe et que c'est un département français (La Réunion) et que beaucoup plus de mots finissent par .re que avec d'autres extensions (j'ai essayé avec .fr ^^). Ça permettrais en outre de faire une astuce à la "youtu.be" ->[.re](https://fr.wikipedia.org/wiki/.re)

J'ai un peu galéré, mais j'ai trouvé un résultat :

**MINOTAURE Is a Nice Online Tale for Adventurers in an Unequal Roleplaying Engine**

Ce qu'on pourrait traduire par : "MINOTAURE est un sympathique récit en ligne pour des aventuriers dans un moteur de jeu de rôle inégal" (quasi textuellement depuis deepL)

L’acronyme récursif n'est pas un exercice toujours heureux, mais j'estime m'en être plutôt bien sorti avec une phrase qui est quasi cohérente en plaçant les principaux thèmes du soft.

**Le nom de domaine [minotau.re](https://shop.gandi.net/fr/domain/suggest?search=minotau.re) est libre**. Et en plus j'ai fait un petit logo (sans prétention), à partir du brainstorm un peu plus haut.

![atrpg_v5](https://user-images.githubusercontent.com/39560791/109322513-af61b680-7852-11eb-8b9f-3c8d36e88b1f.png)

Qu'en pensez vous ?

**Reply by Seikame** [reply]
En ce qui me concerne, cette base de logo c'est un gros oui.

Je suis ni conquis ni rebuté par l'acronyme récursif, je découvre le concept et ça me bouleverse pas des masses. Mais comme j'aime beaucoup l'idée du logo et son .re ça me va. 
Au pire si ça en laisse trop perplexe on pourra changer la/les première lettres de l'acronyme.

**Reply by vouiv-re** [reply]
Damn moi qui pensait avoir fait une réussite critique en charisme grâce à l'acronyme ^^ Blague à part, c'est vrai que l'exercice est un poil trop fastidieux pour un résultat somme toute assez peu transcendant (mais j'en suis quand même fier ^^). Merci de ton retour !

**Reply by uurshin** [reply]
Comme déjà évoqué vocalement, je trouve que Minotaure est un bon nom, et pourrait donner lieu à des visuels sympas.
Le jeu de mot / le nom de domaine est fun, et je trouve le logo très chouette. Merci beaucoup pour ta contribution !
On attend l'avis des autres, et si ça emporte une bonne majorité des suffrages + l'avis et l'autorisation de FibreTigre on partira là dessus.

**Reply by cell** [reply]
D'expérience, un nom de domaine est un peu sensible car il faut le payer. Ca sous-entend qu'une personne va devoir payer chaque année et aura plus de poids que les autres si la vie du jeu. Je conseillerais plus de tenter d'utiliser les github pages.

**Reply by uurshin** [reply]
C'est pas (que) pour faire un portail c'est surtout pour héberger des parties @cell

**Reply by uurshin** [reply]
J'ai contacté FibreTigre pour lui demander son avis sur le changement de nom, et j'ai lié ta proposition, il est chaud !
@HappyFR et @ElethornGM vous en pensez quoi ? @twaiiiin ?

**Reply by HappyFR** [reply]
Perso j'aime bien aussi!
Ma seule préoccupation c'est que c'est pas vraiment explicite, et que sans le slogan, impossible de comprendre le concept (en disant juste Minotaure). Après j'ai pas vraiment de solution à proposer, c'est pas évident haha.

**Reply by vouiv-re** [reply]
Je suis d'accord, mais ATRPG, en plus d'être difficilement prononçable, n'est pas très explicite non plus sans slogan. Au mieux on arrive à percevoir à la première écoute le RPG, mais rien n'est moins sur.
J'ajouterais que le fait de prendre un nom commun est plutôt un avantage car non seulement il se retient bien mais en plus il nous protège partiellement d’éventuels attaques sur la déposition de nom.

Est-ce que j'ai l'air de défendre ma proposition ? Noooon ^^

**Reply by twaiiiin** [reply]
> @HappyFR et @ElethornGM vous en pensez quoi ? @twaiiiin ?

Le nom MINOTAURE est sympa. Le logo est sympa. Pourtant je ne suis pas très emballé par l'acronyme récursif. C'est "amusant" mais comme @HappyFR, je ne trouve pas ça très explicite et évocateur. Mais moi non plus, je n'ai pas vraiment d'autre chose à proposer. Au final mon avis compte assez peu...

A propos du nom de domaine, je rejoins @cell. Si le but est de créer un site et une documentation officielle pour le moteur de jeu, il est préférable d'utiliser Github Pages. Si le but est d'héberger des parties (plus ou moins en mode SaaS), la solution avec un hébergement et un domaine est préférable (mais je ne sais pas si l'attacher au nom du moteur est une très bonne idée. A voir...) Mais comme le dit @cell, cela pose la question de qui le gère, qui le paye, etc. C'est peut-être aller "trop loin" pour "si peu" mais une Association loi de 1901 s'y prêterai peut-être bien si c'est une initiative collective de plusieurs rolistes et streamers.

**Reply by HappyFR** [reply]
J'ai montré le logo à des gens pour tester, et ont m'a dit que ça leur faisait penser à Rodolphe le renne de noël. Surement à cause des cornes et du micro qui fait nez rouge. 
![red-nosed-reindeer-cartoon-illustration-rudolph-white-background-35469281](https://user-images.githubusercontent.com/52490678/109394075-63337680-7925-11eb-8769-2ada419398f0.jpg)

J'ai quelques pistes aussi, en me disant que ce serait bien de mettre en avant le coté coopératif et online, avec des flèches de souris qui vont faire un choix par exemple. 
![Fichier_000](https://user-images.githubusercontent.com/52490678/109393756-dd62fb80-7923-11eb-9747-aaf4e775b610.jpeg)
Je pense qu'il faut faire attention de pas vouloir rajouter tout les éléments dans le logo avec le dès+le mic+la forme du minotaure.

Et pour le nom comme dit avant j'ai rien contre Minotaure, mais j'avais aussi pensé à Tous Pour Un, pour appuyer encore le coté ensemble/choix/aventure/héros par exemple. 

Et aussi un logo n'est pas obligé de prendre au pied de la lettre le titre en l'incarnant forcément. On pourrait très bien avoir Minotaure sans pour autant avoir un minotaure. 
Quand j'y réfléchis, le plus important ce sera la tagline, comment raconter le système le plus clairement possible. Par exemple "jeu de rôle massivement coopératif.

**Reply by vouiv-re** [reply]
Je suis d'accord la tagline est importante !

**Comment by florentcollect** [comment]
Un .re sera indexé très bas dans Google.

**Reply by vouiv-re** [reply]
Je ne suis pas sur qu'on table sur le référencement pour communiquer sur le projet, mais j'avoue que je suis un peu naze sur ce sujet. Quelles pourraient être les conséquences de cette mauvaise indexation ?

**Comment by florentcollect** [comment]
J'avoue que sans JDR dans le nom ca me parle moins à moi. J'ai un peu gambergé pour d'éventuels noms et au final il me reste 

**Ludocratie JDR** (pronomcé ludocracie)/ **Ludocratie RPG** 
Je préfère ludocratie JDR si c'est a destination que des francophones. 

On joue sur ludique (jeu) et démocratie (on joue à base de sondage)... Probablement que la contraction et l'habitude donnerait juste "On fera une partie de ludocratie..."

Ah et pour préciser je suis Fletch (Geek Powa) [https://youtu.be/R8Ojug7IKk4]

**Reply by twaiiiin** [reply]
**Ludocratie JDR** est une très bonne idée, je trouve ! 👍

**Reply by vouiv-re** [reply]
Hello,
Je trouve la piste intéressante ! J'aurais peut-être deux remarques :
* La première est assez anecdotique : Le pouvoir du traître mais surtout celui du leader entre en opposition avec le concept de démocratie, puisque c'est le choix d'un seul plutôt que celui de l'ensemble des joueurs. Alors à priori le rôle du leader sera amené à évoluer ce qui tempère mon propos ^^
* La seconde remarque c'est que le mot "ludocratie" signifie, si je ne m'abuse, "le pouvoir du jeu" et même si le concept est intéressant ne vaudrait-il pas mieux parler du pouvoir des joueurs ?

En tout cas c'est bien on commence à avoir plusieurs propositions, et rien de telle pour prendre une bonne décision !

**Reply by twaiiiin** [reply]
L'idée du suffixe _cratie_ (qui sert à former un nom en rapport avec un pouvoir) est bonne. Cela peut être le pouvoir de tous (démocratie) ou celui d'un seul leader (autocratie). Cela représente plutôt bien le principe de choix des joueurs qui oriente le récit du maître du jeu.

Mais je suis d'accord avec @vouiv-re, le préfixe _ludo_ fait plus référence au jeu (ludique, ludothèque, etc) qu'aux joueurs, et à mon sens davantage au jeu "traditionnel" qu'au jeu de rôle ou au jeu vidéo sur une plateforme de streaming. Je sais pas si c'est vraiment un problème. En tout cas, c'est à creuser...

**Comment by ElethornGM** [comment]
Minotau.re c'est chouette et catchy. 
Le logo réalisé par vouivre est vraiment très cool je trouve !

Je préfèrerais un truc court et facile à prononcer, c'est plus simple en live (et pour ça Minotaure c'est bien !).
Ludocratie RPG ou Ludocratie JDR je trouve ça un peu long à prononcer perso, même si l'idée derrière est plutôt bonne.
On pourrait reprendre le terme de Ludocratie dans la description, du style : 
"Minotau.re - Une ludocratie pour du JDR en streaming interactif"

Je pense qu'il faudrait se mettre une date butoir pour décider d'un nom et d'un début d'identité graphique.

**Comment by uurshin** [comment]
Par avance désolé si ce post est très directif, mais je connais trop bien les situations de blocage sur ces sujets et je préfère profiter de mon énergie et de mon temps quand ils sont disponible plutôt que d'attendre un truc qui piétine.

Si je compte bien, on a : 
- moi, @vouiv-re , @ElethornGM , @Seikame -> plutôt pour Minotau.re
- @florentcollect , @twaiiiin -> plutôt pour Ludocratie
- @cell , @HappyFR -> neutre

Ce n'est pas non plus une décision qui va changer le monde, donc je tranche sur la question pour avancer : on va partir sur Minotaure.

Je prends en charge les frais du nom de domaine minotau.re (5,99 € par an). 
Il y a actuellement une collecte pour l'hébergement de la plateforme (hébergée sur atrpg.fr) que je mets à disposition pour les streamers qui sont dessus, je rajouterai 5,99 au max (sans aucune obligation de participer).

Ça me permet d'en profiter pour utiliser une techno que je connais pas (un cms flat file appelé "GRAV") pour faire le site. Ce site permettra de référencer tous les streams ATRPG (qu'ils soient ou non hébergés sur la plateforme, comme celui de Geek Powa).
Je vous en dis plus au fur et à mesure.

**Reply by ElethornGM** [reply]
D'ailleurs as-tu mis sur le github un lien vers la collecte ? Ou ça n'a pas sa place ici puisque toute la commu n'utilises pas l'hébergement ?
Et donc maintenant on dit les Streams Minotaure, pas les streams ATRPG :) ?

**Reply by uurshin** [reply]
Je mettrai le lien plus tard avec toutes les explications.

Ce serait bien oui ! J'ai commencé à faire un site ici : https://minotau.re mais il faut réfléchir aux textes ensemble.

**Comment by florentcollect** [comment]
Nikel, j'en parlerai dans les actus de la semaine dans le live de geek
powa. Si tu fais un bouton don par paypal sur le site ça peut être pas mal.

Le lun. 22 mars 2021 à 13:14, uurshin ***@***.***> a écrit :

> Ce serait bien oui ! J'ai commencé à faire un site ici :
> https://minotau.re mais il faut réfléchir aux textes ensemble.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fibreville/atrpg/discussions/83#discussioncomment-512863>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATAIS4DCEN4T5MABROVKP6TTE4YD7ANCNFSM4YCMSRJA>
> .
>


-- 
*Florent Chevalier*
Formateur Référencement & Webmarketing

Des formations numériques pour une montée en compétence vertueuse
http://www.collectif-num.fr/   -   📱 06 09 94 69 46
